manifests/011-prometheus-rules.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    name: samples-operator-alerts
+  name: samples-operator-alerts
+  namespace: openshift-cluster-samples-operator
+spec:
+  groups:
+  - name: SamplesOperator
+    rules:
+    - alert: SampleImageStreamImportsFailing
+      expr: max_over_time(sum(openshift_samples_failed_imagestream_import_total)[2h]) > 0
+      annotations:
+        message: |
+          Sample ImageStream imports are consistenly failing over 2 hours.
+    - alert: SamplesDegraded
+      expr: samples_operator_status_degraded == 1
+      annotations:
+        message: |
+          Samples could not be deployed and are in Degraded status.
+    - alert: SamplesInvalidConfig
+      expr: samples_operator_status_invalidconfig == 1
+      annotations:
+        message: |
+          Samples operator has been given an invalid configuration.
+    - alert: SamplesMissingSecret
+      expr: openshift_samples_status_invalidsecret{reason="missing-secret"} == 1
+      annotations:
+        message: |
+          Samples operator cannot find the samples pull secret in the openshift namespace.
```

Comment:
Is "in the openshift namespace" implied? (do we drop it?)

Reply:
given the fact that samples operator copies the install pull secret to the openshift namespace, I thought the clarification was a smidge justified .... I'd rather err on not assuming implications

Reply:
+1

Reply:
+1 to keeping it, that is.  i definitely lean heavy on the "be explicit" side.